restcomm/restcomm.rvd/src/main/webapp/languages/es.json

```diff
@@ -457,7 +458,7 @@
 	"ussdCollectStepContinueToLabel": "Continuar a",
 	"ussdCollectStepContinueToErrorLabel": "Ningún destino especificado",
 	"ussdCollectStepAddOptionButton": "Agregar opción",
-	"ussdCollectStepAssignLabel": "Aignar respuesta a",
+	"ussdCollectStepAssignLabel": "Asignar respuesta a",
 	"ussdCollectStepAssignInputPlaceholder": "tipo el nombre de variable",

```

Comment:
"ussdCollectStepAssignInputPlaceholder": "tipo el nombre de variable", does not make sense. Rephrase or fix.